static/js/components/Header.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+import React from "react"
+
+import { Link } from "react-router-dom"
+
+import { routes } from "../lib/urls"
+
+const Header = () => (
+  <div>
+    <Link to={routes.login}>Login</Link>
```

Comment:
I'd probably say wait until we know what this should look like. I'd also like to address moving away from a global `SETTINGS` var and instead load that data from an endpoint because right now the only way to refresh that object is to forcibly reload the entire page, which isn't great. I think that probably makes it worthy of a PR of itself, so I'll log an issue for it now.

Reply:
#73